sample/benchmark/src/main/java/com/datadog/benchmark/sample/MainActivity.kt

```diff
@@ -21,10 +25,24 @@ class MainActivity : AppCompatActivity() {
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         setContentView(R.layout.activity_main)
+        enableSessionReplay()
+    }
+
+    private fun enableSessionReplay() {
+        val sessionReplayConfig = SessionReplayConfiguration
+            .Builder(SAMPLE_IN_ALL_SESSIONS)
+            .setPrivacy(SessionReplayPrivacy.ALLOW)
```

Comment:
I think we need to think about how parameterise this part. From my previous experiences using the `MASK` option on text views with large content it is very costly on the CPU and can have an impact on performances. We need to be able to test for all the privacy levels.

Reply:
Good point, I will discuss it with the team, and later we can have different run config to test all the privacy strategy.